packages/design-picker/src/components/pattern-assembler-cta/style.scss

```diff
@@ -0,0 +1,26 @@
+.pattern-assembler-cta-wrapper {
```

Comment:
We have to hide it on mobile until we have the mobile version of the Pattern Assembler